src/cmd/tools/dtest/docker/harness/query_api_test.go

```diff
@@ -55,7 +57,9 @@ func TestInvalidInstantQueryReturns400(t *testing.T) {
 func TestInvalidRangeQueryReturns400(t *testing.T) {
 	coord := singleDBNodeDockerResources.Coordinator()
 
-	queryBadRequestTest := []urlTest{
+	urlPrefix := []string{"", "prometheus/", "m3query/"}
+
+	queryBadRequestTest := addPrefixes(urlPrefix, []urlTest{
```

Comment:
I think these two tests only differ by their test table contents and so they can be safely merged. Having two separate test methods made me try figuring out what is the principal difference between them.

Reply:
https://github.com/m3db/m3/pull/2916/commits/661c9ae35af62bd81232983341df94f762e1f6c0#diff-391a291f1f91098bc731847f8843207e5327fe87a52a4599a45ac0cf4c5732e7R69-R75